keras_cv/models/resnet_v1.py

```diff
@@ -55,6 +56,7 @@
 }
 
 BN_AXIS = 3
+BN_EPSILON = 1.001e-5
 
 BASE_DOCSTRING = """Instantiates the {name} architecture.
```

Comment:
#1401 also made some improvements to this base docstring -- would you mind copying those here too?